IPython/core/magics/execution.py

```diff
@@ -71,23 +72,27 @@ class TimeitResult(object):
 
     """
 
-    def __init__(self, loops, repeat, best, worst, all_runs, compile_time, precision):
+    def __init__(self, loops, repeat, average, stdev, all_runs, compile_time, precision):
         self.loops = loops
         self.repeat = repeat
-        self.best = best
-        self.worst = worst
+        self.average = average
+        self.stdev = stdev
```

Comment:
I would do them as simple properties or methods. Caching the values feels like very premature optimisation. This code isn't performance critical, and it's only dealing with a handful of numbers, so even with a Python loop, it will be totally instantaneous from a user perspective.